### ISSUE TITLE
Join https://github.com/plantuml-stdlib

### ISSUE THREAD

**Issue by RicardoNiepel** [dcasati/kubernetes-PlantUML#6]
Hey @dcasati - think about joining the https://github.com/plantuml-stdlib org :)

**Comment by dcasati** [comment]
It’s @RicardoNiepel ! Amazing ! Will do !

**Comment by Potherca** [comment]
@dcasati I've sent you an invitation to join [the plantuml-stdlib org](https://github.com/plantuml-stdlib). Welcome on board!

Once you have accepted, you can [migrate this repository to the `plantuml-stdlib` org](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/transferring-a-repository#transferring-a-repository-owned-by-your-user-account).

Any links in the code or docs here will need to be updated to the repository's new home.
If you do not have the time or energy to change the links, I can do that for you, just let me know.

**Comment by bjet007** [comment]
Any update to integrate this lib into the plantuml-stdlib?

**Comment by Potherca** [comment]
@dcasati  This is your friendly one-year-later reminder to ask if you might be interested.

**Comment by mueller-ma** [comment]
@Potherca Have you thought about forking this repo to https://github.com/plantuml-stdlib? You will lose the open issues and PRs, but that are only 4 issues (including this) and the PRs can be cherry-picked.

**Comment by Potherca** [comment]
Yes, we have been thinking about it, but currently there is [not enough bandwidth][1] to include this (and some other mostly abandoned) repo.

However, I have created this discussion as a means to gather opinions and track interest: https://github.com/orgs/plantuml-stdlib/discussions/6

[1]: https://github.com/orgs/plantuml-stdlib/projects/3/

**Comment by mueller-ma** [comment]
> https://github.com/orgs/plantuml-stdlib/discussions/6

That's a 404 for me. Is it private or the link broken?

**Comment by Potherca** [comment]
Looks like the related repository was set to `private`. Should be visible now...
